pkg/local-storage/member/controller/scheduler/scheduler.go

```diff
@@ -71,9 +75,190 @@ func (s *scheduler) GetNodeCandidates(vols []*apisv1alpha1.LocalVolume) (qualifi
 		}
 	}
 
+	nodes, err := s.filterNodeByAffinity(vols[0], qualifiedNodes)
```

Comment:
why only `vols[0]` is used to filter? this may cause some error in following sceniors:

* **the first schedule is fail and re-schedule comes in**: this may cause different results due to `no-sorted vols array`.
* **different accessmode between two volumes**: one volume is RWO and another is RWO but accessed by many pods on the same node. when the second volume is used at another pod, the node affinity is missing from the relevant pvc and finally cause a scheduler error.

Reply:
Because the pvc affinity under the same pod is inherited to this pod, the affinity is the same, and only one need to be judged